frontend/src/employee-frontend/components/messages/MessageEditor.tsx

```diff
@@ -196,9 +216,27 @@ export default React.memo(function MessageEditor({
       draftContent?.recipientIds
     )
   )
+  const [filtersVisible, setFiltersVisible] = useState(false)
+  const [yearOfBirthTree, setYearOfBirthTree] = useState<TreeNode[]>(
+    range(0, 7).map<TreeNode>((n) => ({
+      text: new Date().getFullYear() - n + '',
```

Comment:
If you need today's date, use `LocalDate.todayInSystemTz()` or `LocalDate.todayInHelsinkiTz()` depending on which you want. Those functions work with our mocked time mechanism and make it explicit which time zone you want to use (browser vs Helsinki). Normal JS Date ignores mocked time, and always uses the browser time zone behind the scenes.